### PR TITLE
Feature/pcs aatf comparison report

### PR DESCRIPTION
**Pull request by SumeN**
Added tests and updated Procedure

### CODE REVIEW

File: src/EA.Weee.RequestHandlers.Tests.Unit/Admin/AatfReports/GetPcsAatfComparisonDataHandlerTests.cs [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            data.FileContent.Should().Contain($"{csvData1.ComplianceYear},{csvData1.QuarterValue},{csvData1.ObligationType},{csvData1.Category},{csvData1.SchemeNameValue},{csvData1.PcsApprovalNumber},{csvData1.PcsAbbreviation},{csvData1.AatfName},{csvData1.AatfApprovalNumber},{csvData1.AatfAbbreviation},{csvData1.PcsTonnage},{csvData1.AatfTonnage},{csvData1.DifferenceTonnage}");
+            data.FileContent.Should().Contain($"{csvData2.ComplianceYear},{csvData2.QuarterValue},{csvData2.ObligationType},{csvData2.Category},{csvData2.SchemeNameValue},{csvData2.PcsApprovalNumber},{csvData2.PcsAbbreviation},{csvData2.AatfName},{csvData2.AatfApprovalNumber},{csvData2.AatfAbbreviation},{csvData2.PcsTonnage},{csvData2.AatfTonnage},{csvData2.DifferenceTonnage}");
+            data.FileContent.Should().Contain($"{csvData3.ComplianceYear},{csvData3.QuarterValue},{csvData3.ObligationType},{csvData3.Category},{csvData3.SchemeNameValue},{csvData3.PcsApprovalNumber},{csvData3.PcsAbbreviation},{csvData3.AatfName},{csvData3.AatfApprovalNumber},{csvData3.AatfAbbreviation},{csvData3.PcsTonnage},{csvData3.AatfTonnage},{csvData3.DifferenceTonnage}");
+        }
+
+        [Fact]
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.RequestHandlers.Tests.Unit/Admin/AatfReports/GetPcsAatfComparisonDataHandlerTests.cs]: Add we add test to make sure that the file name is correct when the quarter and obligation type parameters are not supplied

File: src/EA.Weee.Web/Areas/Admin/Controllers/ReportsController.cs [hunk 2]
```diff
@@ -130,6 +130,9 @@ public ActionResult ChooseReport(ChooseReportViewModel model)
                 case Reports.AatfAeDetails:
                     return RedirectToAction("AatfAeDetails", "AatfReports");
 
+                case Reports.PcsAatfDataDifference:
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Admin/Controllers/ReportsController.cs]: Add test to test the redirect

File: src/EA.Weee.RequestHandlers.Tests.Unit/Admin/AatfReports/GetPcsAatfComparisonDataHandlerTests.cs (hunk 1 above)
Comment (SumeN) [reply src/EA.Weee.RequestHandlers.Tests.Unit/Admin/AatfReports/GetPcsAatfComparisonDataHandlerTests.cs]: Added

File: src/EA.Weee.Web/Areas/Admin/Controllers/ReportsController.cs (hunk 2 above)
Comment (SumeN) [reply src/EA.Weee.Web/Areas/Admin/Controllers/ReportsController.cs]: Added

**Pull request merged**